libs/hdf-converters/src/jsonix-intermediate-converter.ts

```diff
@@ -0,0 +1,9 @@
+import {JsonixConverter} from './jsonix-converter';
```

Comment:
move all the jsonix converter documentation you wrote to the saf cli.  it's fine if it's on it's own page over there.

Reply:
This information was created and reviewed by two other developers here: https://github.com/mitre/heimdall2/wiki/HDF-Converters-How-Tos